bundles/org.openhab.binding.ahawastecollection/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ahawastecollection"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="collectionSchedule">
+		<label>aha Waste Collection Schedule</label>
+		<description>aha Waste Collection Schedule from http://www.aha-region.de/abholtermine/abfuhrkalender</description>
+
+		<channels>
+			<channel id="generalWaste" typeId="collectionDateGeneralWaste"/>
+			<channel id="leightweightPackaging" typeId="collectionDateLeightweightPackaging"/>
+			<channel id="bioWaste" typeId="collectionDateBioWaste"/>
+			<channel id="paper" typeId="collectionDatePaper"/>
+		</channels>
+
+		<config-description>
+			<parameter name="commune" type="text" required="true">
+				<label>Commune</label>
+				<description>Commune from form input</description>
+			</parameter>
+			<parameter name="street" type="text" required="true">
+				<label>Street</label>
+				<description>Street from selected street value. For example "02095@Oesterleystr.+/+Südstadt@Südstadt"</description>
+			</parameter>
+			<parameter name="houseNumber" type="text" required="true">
+				<label>House number</label>
+				<description>House number from form input</description>
+			</parameter>
+			<parameter name="houseNumberAddon" type="text" required="false">
+				<label>House number addon from form input, may be empty</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions